src/styl/custom_theme.styl

```diff
@@ -135,6 +135,17 @@
       padding-right 10px
       padding-left 10px
 
+  .home
+    >.homeCreate
+      >.newGame-pools
+        >.newGame-pool
+          background-color transpWhite005
+          &.active
+            background-color transpWhite01
+  .ceval-pv
```

Comment:
I don't get the need for this change. Please only change home screen style, since it's a PR that is home related